editor/plugins/gizmos/visible_on_screen_notifier_3d_gizmo_plugin.cpp

```diff
@@ -64,11 +64,11 @@ String VisibleOnScreenNotifier3DGizmoPlugin::get_handle_name(const EditorNode3DG
 		case 2:
 			return "Size Z";
 		case 3:
-			return "Pos X";
+			return "Position X";
 		case 4:
-			return "Pos Y";
+			return "Position Y";
 		case 5:
-			return "Pos Z";
+			return "Position Z";
```

Comment:
All of these are not going through `TTR()`, is this normal?

Reply:
Good catch, these should be using `TTR()`.